### PR TITLE
[squid][dmfg] Add cephadm conf and suite

### PR DESCRIPTION
**Pull request by vinpapnoi**
# Description

Add cephadm conf and suite for squid.

### CODE REVIEW

**Review by MohitBis — DISMISSED** [review]
LGTM

File: suites/squid/cephadm/tier-0.yaml [hunk 1]
```diff
@@ -0,0 +1,145 @@
+#===============================================================================================
```
Comment (vamahaja) [inline suites/squid/cephadm/tier-0.yaml:1]: Change test suite name to specific to test suite operations

File: conf/squid/cephadm/tier-0.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+globals:
```
Comment (vamahaja) [inline conf/squid/cephadm/tier-0.yaml:1]: Change name with cluster node details

File: conf/squid/cephadm/tier-1_9node_mix_client.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
+
```
Comment (vamahaja) [inline conf/squid/cephadm/tier-1_9node_mix_client.yaml:1]: Same as above

File: suites/squid/cephadm/tier-2_rhosd_multiple_subnet_openstack_only.yaml [hunk 4]
```diff
@@ -0,0 +1,109 @@
+# ==================================================================================
```
Comment (vamahaja) [inline suites/squid/cephadm/tier-2_rhosd_multiple_subnet_openstack_only.yaml:1]: Use `tier2`

Comment (vamahaja) [reply suites/squid/cephadm/tier-2_rhosd_multiple_subnet_openstack_only.yaml:1]: Update suites names as suggested.

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: suites/squid/cephadm/tier-0.yaml (hunk 1 above)
Comment (vinpapnoi) [reply suites/squid/cephadm/tier-0.yaml]: done

File: conf/squid/cephadm/tier-1_9node_mix_client.yaml (hunk 3 above)
Comment (vinpapnoi) [reply conf/squid/cephadm/tier-1_9node_mix_client.yaml]: done

File: suites/squid/cephadm/tier-2_rhosd_multiple_subnet_openstack_only.yaml (hunk 4 above)
Comment (vinpapnoi) [reply suites/squid/cephadm/tier-2_rhosd_multiple_subnet_openstack_only.yaml]: done

File: conf/squid/cephadm/tier-0.yaml (hunk 2 above)
Comment (vinpapnoi) [reply conf/squid/cephadm/tier-0.yaml]: done

File: suites/squid/cephadm/tier0.yaml [hunk 5]
```diff
@@ -0,0 +1,145 @@
+#===============================================================================================
```
Comment (vamahaja) [inline suites/squid/cephadm/tier0.yaml:1]: This test suite is part of smoke tests [1], can be removed from here.
[1] https://github.com/red-hat-storage/cephci/blob/master/suites/squid/smoke/bvt.yaml

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: suites/squid/cephadm/tier0.yaml (hunk 5 above)
Comment (vinpapnoi) [reply suites/squid/cephadm/tier0.yaml]: Done

**Review by vamahaja — APPROVED** [review]

**Pull request merged**